test/readEveryItem-test.js

```diff
@@ -0,0 +1,65 @@
+var FeedSub = require('..')
+  , nock = require('nock')
+  , assert = require('assert')
+  , path = require('path')
+
+
+var file1 = path.join(__dirname, 'assets', 'googlefeed.xml')
+  , file2 = path.join(__dirname, 'assets', 'googlefeedupdated.xml')
+
+
+describe('Read all published/updated items with readEveryItem', function() {
+
+  var host = 'https://www.blogger.com'
+    , path = '/feeds/10861780/posts/default'
+    , reader = new FeedSub(host + path, {
+        emitOnStart: true, readEveryItem: true
+      })
+    , itemCount = 0
+    , itemsEvents = 0
+
+  reader.on('item', function() {
+    itemCount++;
+  });
+
+  reader.on('items', function() {
+    itemsEvents++;
+  });
+
+  nock(host)
+    .get(path)
+    .replyWithFile(200, file1)
+
+  it('Should return all items', function(done) {
+    reader.read(function(err, items) {
+      if (err) throw err;
+      assert.ok(!err);
```

Comment:
I know I have these in my other tests file, but it's not needed since we're already checking if it's set on the previous line. I'll clean them out from the other tests eventually.